Wikimate.php

```diff
@@ -1053,7 +1054,7 @@ public function getFilename()
 	public function getInfo($refresh = false)
 	{
 		if ($refresh) { // We want to query the API
-
+			// Specify all image properties to retrieve
```

Comment:
Shouldn't either this line use "relevant ... properties", or line 529 use "all ... properties"? For consistency.